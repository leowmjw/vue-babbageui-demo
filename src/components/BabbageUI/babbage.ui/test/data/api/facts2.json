{
  "fields": [
    "approved",
    "adjusted",
    "executed",
    "administrative_classification_admin1.admin1"
  ],
  "cell": [
  ],
  "page": 1,
  "status": "ok",
  "total_fact_count": 190525,
  "order": [
  ],
  "data": [
    {
      "executed": 0.0,
      "administrative_classification_admin1.admin1": "Central",
      "approved": 100,
      "adjusted": 200
    },
    {
      "executed": 5,
      "administrative_classification_admin1.admin1": "Central",
      "approved": 10,
      "adjusted": 20
    },
    {
      "executed": 10,
      "administrative_classification_admin1.admin1": "Central",
      "approved": 20,
      "adjusted": 0
    },
    {
      "executed": 0.0,
      "administrative_classification_admin1.admin1": "Central",
      "approved": 0,
      "adjusted": 0
    },
    {
      "executed": 1,
      "administrative_classification_admin1.admin1": "Other",
      "approved": 2,
      "adjusted": 1
    }
  ],
  "page_size": 5
}