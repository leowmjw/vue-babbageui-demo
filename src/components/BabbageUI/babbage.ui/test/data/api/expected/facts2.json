{
  "headers": [
    {
      "key": "approved",
      "label": "Approved",
      "numeric": true
    },
    {
      "key": "adjusted",
      "label": "Adjusted",
      "numeric": true
    },
    {
      "key": "executed",
      "label": "Executed",
      "numeric": true
    },
    {
      "key": "administrative_classification_admin1.admin1",
      "label": "Admin1",
      "numeric": false
    }
  ],
  "columns": [
    [
      100,
      200,
      0,
      "Central"
    ],
    [
      10,
      20,
      5,
      "Central"
    ],
    [
      20,
      0,
      10,
      "Central"
    ],
    [
      0,
      0,
      0,
      "Central"
    ],
    [
      2,
      1,
      1,
      "Other"
    ]
  ],
  "info": {
    "total": 190525,
    "pageSize": 5,
    "page": 1
  }
}